app/models/import/heat_source_builder.rb

```diff
@@ -2,7 +2,7 @@ class Import
   module HeatSourceBuilder
     NumberOfUnitsAttribute     = Attribute.new('number_of_units')
     TechnicalLifetimeAttribute = Attribute.new('technical_lifetime')
-    MarginalCostsAttribute     = Attribute.new('marginal_costs')
+    MarginalCostsAttribute     = Attribute.new('marginal_heat_costs')
```

Comment:
Tiny nitpick, but can we rename this to MarginalHeatCostsAttribute, in case we need marginal_costs later?